.gitignore

```diff
@@ -10,3 +10,6 @@ lodestar-db
 lodestar-config.toml
 keys/
 test-db
+yarn-error.log
+package-lock.json
```

Comment:
you should use yarn though :)

Reply:
`yarn*` this?

Reply:
I mean we need yarn.lock to be in repo. But you got package-lock.json because you ran commands like `npm install`

Reply:
Mistakenly used it once. Generally, I use `yarn install`.